packages/components/psammead-grid/README.md

```diff
@@ -136,19 +213,77 @@ Note that here, any time you use `<Grid>` that generates a new grid. The total n
 import Grid from '@bbc/psammead-grid';
 
 const MyComponent = () => (
-  <Grid columns={{ group2: 6, group3: 6 }}>
-    <Grid columns={{ group2: 6, group3: 6 }}>
-      <Grid item columns={{ group2: 6, group3: 3 }}>
+  <Grid
+    columns={{
+      group0: 4,
```

Comment:
To ensure this example is easier to follow, please keep these columns having the same values. e.g. `group0: 6, group1: 6, group2: 6, group3: 6, group4: 6, group5: 6`. That way, if someone was to use this example, the output would look just like the screenshot shown below: https://github.com/bbc/psammead/pull/2458/files#diff-afe1f4233782372163743bf32a6bc5eeR294